bricks/vs/src/code-editor/utils/isInEmbedded.ts

```
@@ -0,0 +1,69 @@
+  if (
+    previousEnd &&
+    previousStart.range
+      .getEndPosition()
+      .isBefore(previousEnd.range.getStartPosition())
+  ) {
+    return false;
+  }
+  const nextStart = findNextMatch(model, "<%", position);
+  if (
+    nextStart &&
+    nextStart.range.getStartPosition().isBefore(nextEnd.range.getEndPosition())
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)